distribution/lib/Standard/Table/0.0.0-dev/src/Excel/Excel_Format.enso

```diff
@@ -69,4 +72,11 @@ type Excel_Format
         case self.section of
             Excel_Section.Sheet_Names -> Error.throw (Illegal_Argument.Error "Sheet_Names cannot be used for `write`.")
             Excel_Section.Range_Names -> Error.throw (Illegal_Argument.Error "Range_Names cannot be used for `write`.")
+            Excel_Section.Workbook ->
+                sheet_name = if file.exists.not then "Sheet1" else
+                    names = Excel_Reader.read_file file Excel_Section.Sheet_Names False on_problems format
+                    name_map = Map.from_vector (names.map n-> [n, 1])
+                    idx = 1.up_to 256 . find n-> (name_map.contains_key ("Sheet" + n.to_text) . not)
+                    "Sheet" + idx.to_text
```

Comment:
What if there are already 256 sheets in the file?

I assume this is a limit of Excel?

We will currently get a `Not_Found` error. I think we should get a more specific error here - something saying 'write failed because a new sheet cannot be added to the target file as it reached the limit.'